extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/runtime/RedisConfig.java

```diff
@@ -150,5 +158,30 @@
          */
         @ConfigItem(defaultValue = "32")
         public int maxNestedArrays;
+
+        /**
+         * The number of reconnection attempts when a pooled connection cannot be established on first try.
+         */
+        @ConfigItem(defaultValue = "0")
+        public int reconnectAttempts = 0;
+
+        /**
+         * The interval between reconnection attempts when a pooled connection cannot be established on first try.
+         */
+        @ConfigItem(defaultValue = "PT1S")
+        public Duration reconnectInterval = Duration.ofSeconds(1L);
+
+        /**
+         * The maximum time a connection remains unused in the pool before it is closed.
+         */
+        @ConfigItem(defaultValueDocumentation = "no timeout")
+        public Optional<Integer> idleTimeout = Optional.empty();
+
+        /**
+         * SSL/TLS config.
+         */
+        @ConfigItem
+        public Optional<SslConfig> ssl = Optional.empty();
```

Comment:
```suggestion
        public SslConfig> ssl;
```

No need for this to be optional since it can be enabled/disabled and it has a default value.